### PR TITLE
updated 03 lesson and extras pages

### PR DESCRIPTION
**Pull request by tracykteal**
updated 03 lesson and extras pages

### CODE REVIEW

File: _episodes/03-ncbi-sra.md [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+- "Be aware that public genomic data is available"
+- "Understand how to access and download this data"  
 keypoints:
-- "First key point."
+- ""
 ---
 
+In our experiments we're usually generating our own genomic data, but many types of analyses use reference data or you may want to use it to compare your results or annotate your data with publicly available data. You may also want to do a full project or set of analyses using publicly available data. This data is a great, and essential, resource for genomic data analysis.
+
+There are many repositories for public data. Some model organisms or fields have specific databases, and there are ones for particular types of data. Two of the most comprehensive are the [National Center for Biotechnology Information  (NCBI)](https://www.ncbi.nlm.nih.gov) and [European Nucleotide Archive < EMBL-EBI](https://www.ebi.ac.uk/ena). In this lesson we're working with the NCBI database, but the general process is the same for any database.
```
Comment (ErinBecker) [inline _episodes/03-ncbi-sra.md]: Should EMBL-EBI be in parens for similarity with how NCBI is referenced in the previous line?

File: _episodes/03-ncbi-sra.md [hunk 2]
```diff
@@ -1,17 +1,22 @@
...
+The [sequencing dataset we'll be using in this lesson](http://www.datacarpentry.org/organization-genomics/extras/data.html) was obtained from the [NCBI Sequence Read Archive](http://www.ncbi.nlm.nih.gov/sra), which is a large (>3 quadrillion basepairs as of 2014) repository for next-generation sequence data. Like many NCBI databases, it is complex and mastering its use is greater than the scope of this lesson. Very often, as in the Lenski paper, there will be a direct link (perhaps in the supplemental information) to where on the SRA the dataset can be found. The link from the Lenski paper is: [http://www.ncbi.nlm.nih.gov/sra?term=SRA026813](http://www.ncbi.nlm.nih.gov/sra?term=SRA026813)
```
Comment (ErinBecker) [inline _episodes/03-ncbi-sra.md]: Where does this data.html page go?

Comment (ErinBecker) [reply _episodes/03-ncbi-sra.md]: Should we add a line saying that we've done some modifications of that data for this lesson?

File: _extras/guide.md [hunk 3]
```diff
@@ -3,4 +3,10 @@ layout: page
 title: "Instructor Notes"
 permalink: /guide/
 ---
-FIXME
+
+This lesson has many exercises as discussions to prompt people to think about
+the components of their genomics project and the workflow.
+
+Discussions can happen between neighbors in a workshop. Then after paired discussion there can be a short general discussion of the types of things that came up in the discussion.
+
```
Comment (ErinBecker) [inline _extras/guide.md:11]: Etherpad should be capitalized.

**Review by ErinBecker — CHANGES_REQUESTED** [review]

**Comment by ErinBecker** [comment]
LGTM with a few minor modifications. Please go ahead and merge with or without those changes.

**Comment by tracykteal** [comment]
Updated with @ErinBecker suggestions.

**Pull request merged**